### PR TITLE
Admin: Add carts module for monitoring abandoned carts

### PR DESCRIPTION
**Pull request by shawnadelic**
Add an admin carts module that allows merchants to monitor abanded
user carts, total sales, date since last update, and other
useful information.

Refs SHOOP-2237

### CODE REVIEW

**Comment by tulimaki** [comment]
This admin module should live under front since stored basket is under front and this will break the admin if the front is not in installed apps.

File: shoop/admin/modules/carts/views/list.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        ),
+        Column(
+            "taxful_total_price", _(u"Total"), sort_field="taxful_total_price_value",
+            display="format_taxful_total_price", class_name="text-right",
+            filter_config=RangeFilter(field_type="number", filter_field="taxful_total_price_value")
+        ),
+    ]
+
+    def format_order_date(self, instance, *args, **kwargs):
+        return format_datetime(localtime(instance.created_on), locale=get_current_babel_locale())
```
Comment (tulimaki) [inline shoop/admin/modules/carts/views/list.py]: This formatting should probably be in ``shoop.utils.dates`` as an util.

**Comment by shawnadelic** [comment]
test this please

**Comment by tulimaki** [comment]
Can you move all front admin-module related code under fron/admin_module/?

File: doc/changelog.rst [hunk 2]
```diff
@@ -66,6 +66,7 @@ Front
 - Remove ``get_method_validation_errors`` signal
 - Fix bug at ``get_visible_products`` filter when orderable_only is False
 - Set template price display options from the customer
+- Add admin view for monitoring customer carts
```
Comment (suutari-ai) [inline doc/changelog.rst]: Move this entry to top, since it's newest.

**Comment by suutari-ai** [comment]
Yeah, the admin module stuff should be under shoop/front/admin_modules, since now it's very confusing and hard to tell that these views are actually admin views.

Also, it's confusing that now we have used both terms basket and cart for the same thing. Can we do anything about it? I like "cart" more, since that's what we want to show in the UI, but that would need much refactoring.

**Comment by suutari-ai** [comment]
OK the "basket" -> "cart" change ain't that difficult after all... almost got it done in 10 minutes: https://github.com/shoopio/shoop/compare/master...suutari-ai:basket-to-cart  ... just needs the migrations to be fixed... that can be difficult though. Maybe we shouldn't go that way yet...

**Pull request merged**